dist/ps-web-apis.d.ts

```diff
@@ -48,5 +69,49 @@ export interface WhoamiV1 {
      * @param clientId The string identifier of the client for which the customer id is requested.
      */
     getCustomerId(clientId: string): Promise<string>;
+    /**
+     * will provide unsafe purchase data
+     */
+    getUnsafePurchaseData(): Promise<PurchaseData>;
+    /**
+     * will provide jaId
+     */
+    getJaId(): string;
```

Comment:
Can you add an additional commen:. The Id is only available for loggedIn users. Otherwise this function throws an error. May @throws works here in the comment.